### ISSUE TITLE
edit-with-external-editor should remember the focused element

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#1350]
Recipe:

- Focus on an element.
- Call `edit-with-external-editor`.
- While the editor is open, go back to Nyxt an focus on another element.
- Close the editor.

Result: the editor content is pasted in the element (or not at all, if the element is not writable).
Instead, the command should be smarter and remember the focused element and focused back to it when the editor exits.

**Comment by aadcg** [comment]
Indeed. I think you should edit the issue, since it's hard to follow your references to the elements (tho I understood clearly of course).

**Comment by Ambrevar** [comment]
What do you mean?  Replace "element" with something else?

**Comment by aadcg** [comment]
I mean that you reference 2 elements (the one where the edit takes place, and those selected afterwards), and these references aren't always clear.

**Comment by Ambrevar** [comment]
Indeed, I've just edited the original post to clarify this.

**Comment by aadcg** [comment]
Voila, perfect!
